diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
index fdd00658f55fe6e7db0ed107fa9e9341aebf7aca..08a5b17b6cc64e974c137accc65605fb610e3948 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
@@ -430,6 +430,7 @@ void CheckClientDownloadRequestBase::SendRequest() {
   client_download_request_->set_skipped_certificate_whitelist(
       skipped_certificate_whitelist_);
 
+  // BRAVE_SEND_REQUEST_FILTER
   if (!client_download_request_->SerializeToString(
           &client_download_request_data_)) {
     FinishRequest(DownloadCheckResult::UNKNOWN, REASON_INVALID_REQUEST_PROTO);
